selene_sdk/samplers/multi_sampler.py

```diff
@@ -0,0 +1,323 @@
+"""
+This module provides the `MultiSampler` class, which uses a
+FileSampler for each mode of sampling (train, test, validation).
+The MultiSampler is therefore a subclass of Sampler.
+"""
+import numpy as np
+from torch.utils.data import DataLoader
+
+from .sampler import Sampler
+
+def MultiFileSampler(*args, **kwargs):
+    from warnings import warn
+    warn("MultiFileSampler is deprecated and will be removed from future "
```

Comment:
Oh sorry I see this now - actually what I meant was to keep the `MultiFileSampler` file in this release and warn that it's not going to be maintained any longer in the `__init__` but now I see the parameters for `MultiSampler` and `MultiFileSampler` are the same

Reply:
We can do it the way you described too. As you said it's mainly a difference of docs (we can probably make MultiFileSampler generate the same docs too by copying over the doc attribute - depending on what do you prefer!

Reply:
sure, let's copy over the doc attributes and also add the deprecation warning in the docstring description.

Reply:
actually I realized it is not easy to copy over all docs including methods. currently, I wrote a doc for the `MultiFileSampler` indicating that it is deprecated and user should refer to the `MultiSampler` - is this fine?